src/panels/config/integrations/integration-panels/ozw/dialog-ozw-refresh-node.ts

```diff
@@ -0,0 +1,274 @@
+        .heading=${createCloseHeading(
+          this.hass,
+          this.hass.localize("ui.panel.config.ozw.refresh_node.title")
+        )}
+      >
+        ${this._complete
+          ? html`
+              <p>
+                ${this.hass.localize(
+                  "ui.panel.config.ozw.refresh_node.complete"
+                )}
+              </p>
+              <mwc-button slot="primaryAction" @click=${this._close}>
+                ${this.hass.localize("ui.common.close")}
+              </mwc-button>
+            `
+          : html`
+              ${this._active
+                ? html`
+                    <div class="flex-container">
+                      <ha-circular-progress active></ha-circular-progress>
+                      <div>
+                        <p>
+                          <b>
+                            ${this.hass.localize(
+                              "ui.panel.config.ozw.refresh_node.refreshing_description"
+                            )}
+                          </b>
+                        </p>
+                        ${this._node
+                          ? html`
+                              <p>
+                                ${this.hass.localize(
+                                  "ui.panel.config.ozw.refresh_node.node_status"
+                                )}:
+                                ${this._node.node_query_stage}
+                                (${this.hass.localize(
+                                  "ui.panel.config.ozw.refresh_node.step"
+                                )}
+                                ${nodeQueryStages.indexOf(
+                                  this._node.node_query_stage
+                                ) + 1}/17)
+                              </p>
+                              <p>
+                                <em>
+                                  ${this.hass.localize(
+                                    "ui.panel.config.ozw.node_query_stages." +
+                                      this._node.node_query_stage.toLowerCase()
+                                  )}</em
+                                >
+                              </p>
+                            `
+                          : ``}
+                      </div>
+                    </div>
+                  `
+                : html`
+                    ${this.hass.localize(
+                      "ui.panel.config.ozw.refresh_node.description"
+                    )}
+                    <p>
+                      ${this.hass.localize(
+                        "ui.panel.config.ozw.refresh_node.battery_note"
+                      )}
+                    </p>
+                  `}
+              ${this._nodeMetaData?.WakeupHelp !== ""
+                ? html`
+                    <b>
+                      ${this.hass.localize(
+                        "ui.panel.config.ozw.refresh_node.wakeup_header"
+                      )}
+                      ${this._nodeMetaData?.Name}
```

Comment:
We are sure at this point `_nodeMetaData` is defined, so we can change this to:
```suggestion
                      ${this._nodeMetaData!.Name}
```